manager/controllers/app/fybrikapplication_controller.go

```diff
@@ -137,10 +136,8 @@ func (r *FybrikApplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		if result, err := r.reconcile(applicationContext); err != nil {
 			// another attempt will be done
 			// users should be informed in case of errors
-			if !equality.Semantic.DeepEqual(&application.Status, observedStatus) {
-				// ignore an update error, a new reconcile will be made in any case
-				_ = r.Client.Status().Update(ctx, application)
-			}
+			// ignore an update error, a new reconcile will be made in any case
```

Comment:
if we do reconcile/update at any case, why do we need this Update call at all ?

Reply:
If the error does not go away, it will never be propagated to the status.